src/services/lobby.service.ts

```diff
@@ -2,18 +2,26 @@ import {gameRepository} from "../repositories/game.repository";
 import {GameOptions, IGame, IGameCreateSchema} from "../interfaces/game.interface";
 import HttpException from "../application/utils/exceptions/http-exceptions";
 import HttpStatusCode from "../application/utils/exceptions/statusCode";
+import { ITeamCreateSchema } from "../interfaces/team.interface";
+import { teamService } from "./team.service";
 
-const MAX_TEAMS = 10 // TODO: REMOVE THE MAGICAL_INAPPROPRIACY
+class LobbyService 
+{
+  private teamMembers: Map<string, Set<string>> = new Map();
+
+  async createLobby(hostId: string, options: GameOptions): Promise<string> 
+  {
 
-class LobbyService {
-  async createLobby(hostId: string, options: GameOptions): Promise<string> {
     const gameCreateSchema: IGameCreateSchema = {
       hostId: hostId,
       options: options,
       teams: [],
     };
+
     const gameId = await gameRepository.create(gameCreateSchema);
-    if (!gameId) {
+
+    if (!gameId) 
+    {
       throw new HttpException(HttpStatusCode.INTERNAL_SERVER_ERROR, 'Lobby could not be created.');
```

Comment:
so generally in `gameRepository.create()` will be thrown exception and we cant reach this line in case of exception, so I think it will be better to try-catch such places

Reply:
Thank you, for the security reason I added try-catch in each method